spec/user_spec.rb

```diff
@@ -1,10 +1,18 @@
-require './lib/models/user'
+require './lib/models/user.rb'
 
 describe User do
+
+  before do
+    subject = User.new
+    subject.name = "Amber"
+    subject.save
+  end
+
   it 'can create a new user' do
-    user = User.create
-    expect(user).not_to be nil
+    expect(User.last).not_to be nil
```

Comment:
Excellent. Noticed that I wrote this test and it doesn't do anything, so nice catch.